ecs-init/config/common.go

```diff
@@ -93,6 +93,12 @@ const (
 
 	// DefaultRegionEnvVar is the environment variable for specifying the default AWS region to use.
 	DefaultRegionEnvVar = "AWS_DEFAULT_REGION"
+
+	// ECSGMSASupportEnvVar indicates that the gMSA is supported
+	ECSGMSASupportEnvVar = "ECS_GMSA_SUPPORTED"
```

Comment:
it seems like we're introducing 3 new env vars:
`ECS_GMSA_SUPPORTED`
`CREDENTIALS_FETCHER_HOST`
`ECS_DOMAIN_JOINED_LINUX_INSTANCE`

- can we update the README to tell which one is configurable vs which one isnt? i imagine `ECS_DOMAIN_JOINED_LINUX_INSTANCE` shouldn't be configurable?
- can you initialize the `ECS_DOMAIN_JOINED_LINUX_INSTANCE` in this list as well and reuse the var?

Reply:
ECS_GMSA_SUPPORTED is existing variable
CREDENTIALS_FETCHER_HOST - will add to readme
ECS_DOMAIN_JOINED_LINUX_INSTANCE this is not configurable, ecs-init will set this variable

Reply:
added the variable CREDENTIALS_FETCHER_HOST  to readme